<launch>
    <!-- Ensure stdout/stderr is captured as galaxy_camera-*.log -->
    <node name="galaxy_camera" pkg="galaxy_camera" type="galaxy_camera" output="log">
        <param name="image_width" value="4096"/>
        <param name="image_height" value="3000"/>
        <!-- <param name="image_width" value="1920"/>
        <param name="image_height" value="1080"/> -->
        <param name="pixel_format" value="bgr8"/>
        <param name="camera_info_url" value=""/>
        <param name="camera_frame_id" value="camera"/>
        <param name="camera_ip" value="192.168.40.11"/>
        

        <!-- Exposure -->
        <param name="exposure_auto" value="true"/>
        <!-- <param name="exposure_value" value="20000"/>
        <param name="exposure_min" value="20"/>
        <param name="exposure_max" value="1000000"/> -->

        <!-- Gain -->
        <param name="gain_auto" value="true"/>
        <!-- <param name="gain_value" value="15"/>
        <param name="gain_min" value="0"/>
        <param name="gain_max" value="24"/> -->

        <!-- Frame rate -->
        <param name="frame_rate" value="5"/>

        <!-- White balance -->
        <param name="white_auto" value="true"/>
        <!-- 0, 1, 2 for red, green, blue-->
    </node>
</launch>
